tools/downloader/topologies.py

```diff
@@ -0,0 +1,104 @@
+import os
```

Comment:
But this is API for downloader+converter. Why not `tools/downloader/open_model_zoo.py` or at least `tools/open_model_zoo.py`?

Reply:
> But this is API for downloader+converter.

I disagree with this assessment. It is an API for Open Model Zoo, which _uses_ the downloader & converter to provide its functionality.

> Why not tools/downloader/open_model_zoo.py or at least tools/open_model_zoo.py?

Because it's not a tool. It's a library.